src/commands/url.js

```diff
@@ -1,10 +1,26 @@
 import chalk from 'chalk';
+import fp from 'lodash/fp';
 
-import { UrlUtils } from 'xdl';
+import { UrlUtils, Project } from 'xdl';
 
 import log from '../log';
 import urlOpts from '../urlOpts';
 
+const logArtifactUrl = (platform) => async (projectDir, options) => {
+  const res = await Project.buildAsync(projectDir, { current: false, mode: 'status' });
+  const url = fp.compose(
+    fp.get(['artifacts', 'url']),
+    fp.head,
+    fp.filter(job => platform && job.platform === platform),
+    fp.getOr([], 'jobs')
+  )(res);
+  if (url) {
+    console.log(url);
```

Comment:
Using `console.log` because `log` has a prefix (`[exp] [timestamp] <msg>`) that will interfere with outside commands like `curl`.

Reply:
IIRC you can use `log.nested()` to print without the prefix.